src/js/components/Tabs/doc.js

```diff
@@ -90,6 +96,15 @@ export const themeDoc = {
     type: 'string | { dark: string, light: string }',
     defaultValue: undefined,
   },
+  'tabs.header.border': {
```

Comment:
For consistency with Box border, any reason we don't allow the `style` of the border (dash, etc...)

Reply:
No reason to exclude. I had pulled the logic from `Tab` which wasn't including style, but I think it's a valid thing to have since it's included on Box. I will add it here.

Reply:
Style has been added.